app/questions.py

```diff
@@ -25,28 +25,31 @@ def json(self):
 
 
 class Question:
-    def __init__(self, seed):
+    def __init__(self, seed, independent_var):
         self.description = ""
         self.question = ""
         self.seed = seed
+        self.independent_var = independent_var
     
     @property
     def json(self):
         return {
             "description": self.description,
             "question": self.question,
+            "independent_var": self.independent_var,
         }
 
 
 class MCATQuestion1(Question):
-    def __init__(self, seed):
-        super().__init__(seed)
+    def __init__(self, seed, independent_var="x"):
+        super().__init__(seed, independent_var)
 
-        x = symbols("x")
+        var = symbols(self.independent_var)
         a, b = maths_service.integer_coefficients(amount=2, seed=self.seed)
 
         self.description = "Solve a linear equation."
-        self.question = latex(Eq(a * x + b, 0))
+        self.question = latex(Eq(a * var + b, 0))
+        self.independent_var = independent_var
```

Comment:
Do we need this line if we're already doing `super().__init__(...)` earlier?

Reply:
Yes, because `Question` doesn't accept args for `question` or `description` etc (yet)

It raises a good point though about the necessity of either `Question` or `MCATQuestion1`, which will need more thought before we start implementing more questions